spec.bs

```diff
@@ -670,6 +677,30 @@ button.
 documentPictureInPicture.requestWindow({ disallowReturnToOpener: true });
 </pre>
 
+## Disallow position reuse ## {#example-disallow-position-reuse}
```

Comment:
Nit: Maybe change this section title to "Ignore previous window bounds" to reflect new name.

Reply:
done